pentasharp/Controllers/TaxiCompanyController.cs

```diff
@@ -22,6 +23,17 @@ public TaxiCompanyController(AppDbContext context, IMapper mapper)
             _mapper = mapper;
         }
 
+        [HttpGet("GetTaxiStatuses")]
+        public IActionResult GetTaxiStatuses()
+        {

```

Comment:
move the business logic from here to a service, create its interface, register them in program.cs and inject them properly in this controller. Do this for all the methods in this controller so we dont lose time on creating a refactoring ticket. If you have any unclarities Olta already did a similiar fix in this PR https://github.com/starlabspro-internship/pentasharp/pull/24